pandas/tests/series/test_constructors.py

```diff
@@ -733,7 +736,7 @@ def test_constructor_cast(self):
         with pytest.raises(ValueError, match=msg):
             Series(["a", "b", "c"], dtype=float)
 
-    @pytest.mark.xfail(np_version_gte1p24, reason="GH49777")
+    @pytest.mark.xfail(np_version_gte1p24 and not IS64, reason="GH49777")
```

Comment:
The test also fails on 64bit

Reply:
At least for me on openSUSE. I don't know why it does XPASS on your second to last commit.